sdk/monitor/azure-monitor-opentelemetry-exporter/tests/test_base_exporter.py

```diff
@@ -81,49 +87,61 @@ def test_constructor(self):
         self.assertEqual(base._timeout, 10)
         self.assertEqual(base._api_version, "2021-02-10_Preview")
         self.assertEqual(base._storage_min_retry_interval, 100)
+        self.assertEqual(base._storage_path, "test/path")
+
 
+    def test_transmit_from_storage_failed(self):
```

Comment:
If this is the replacement for failed_not_retryable, shouldn't we call it that to be more specific?

Reply:
Actually that is the wrong description. Renamed it to the success case.